File: nomad-samples/python/basic-live-channel-1.1/live_input/live_input_types.py
from constants.system_constants import *

'''
 * Live Input Types
 *
 * "RTP Push"
 * "RTMP Push"
 * "RTMP Pull"
 * "URL Pull"
 *
'''
LIVE_INPUT_TYPES = {
    "RTMP_PULL": RTMP_PULL_INPUT_TYPE_LOOKUP_ID,
    "RTMP_PUSH": RTMP_PUSH_INPUT_TYPE_LOOKUP_ID,
    "RTP_PUSH": RTP_PUSH_INPUT_TYPE_LOOKUP_ID,
    "URL_PULL": URL_PULL_INPUT_TYPE_LOOKUP_ID
}

File: nomad-samples/admin/js/basic-live-channel/live-input/live-input-types.js
import * as sysConstants from "../constants/system-constants.js";

/**
 * Live Input Types
 *
 * "RTP Push"
 * "RTMP Push"
 * "RTMP Pull"
 * "URL Pull"
 *
 */
const liveInputTypes = {
    RTMP_PULL: sysConstants.RTMP_PULL_INPUT_TYPE_LOOKUP_ID,
    RTMP_PUSH: sysConstants.RTMP_PUSH_INPUT_TYPE_LOOKUP_ID,
    RTP_PUSH: sysConstants.RTP_PUSH_INPUT_TYPE_LOOKUP_ID,
    URL_PULL: sysConstants.URL_PULL_INPUT_TYPE_LOOKUP_ID
};

export default liveInputTypes;
